src/Application.php

```diff
@@ -128,8 +128,10 @@ public function services(ContainerInterface $container): void
     protected function bootstrapCli(): void
     {
         try {
+            $this->addPlugin('Cake/Repl');
```

Comment:
Shouldn't this try one plugin at a time?

Reply:
In this case, yes.
The grouping only makes sense if the following can only exist or be available with the first.

Reply:
I can split it up if you'd prefer.

Reply:
A simple `foreach (['Cake/Repl', 'Bake'] as $plugin)` should work.